{"name": "查询用户", "status": "broken", "statusDetails": {"message": "FileNotFoundError: [Errno 2] No such file or directory: '../api/wework.yaml'", "trace": "self = <WechatAPI.testcase.test_work.TestWework object at 0x000001FBD73BDE48>\n\n    @allure.title(\"查询用户\")\n    def test_get_work(self):\n>       print(Wework().test_get_member(\"qw123\"))\n\nWechatAPI\\testcase\\test_work.py:23: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <WechatAPI.api.we_work.Wework object at 0x000001FBD5DBC048>\n\n    def __init__(self):\n        self.token = Util().token() #不要忘了Util的()\n        self.params[\"token\"]=self.token #要记得替换token的变量\n    \n>       with open(\"../api/wework.yaml\",encoding=\"utf-8\")as f:\nE       FileNotFoundError: [Errno 2] No such file or directory: '../api/wework.yaml'\n\nWechatAPI\\api\\we_work.py:27: FileNotFoundError"}, "start": 1647478468596, "stop": 1647478468963, "uuid": "70526b38-999c-4cd8-9e6d-e592cb9f6b23", "historyId": "bd5074aba9b8051b8d023fa8b970968e", "testCaseId": "9d134167012fc710b2b6d7e318243f8c", "fullName": "WechatAPI.testcase.test_work.TestWework#test_get_work", "labels": [{"name": "parentSuite", "value": "WechatAPI.testcase"}, {"name": "suite", "value": "test_work"}, {"name": "subSuite", "value": "TestWework"}, {"name": "host", "value": "ZXJ"}, {"name": "thread", "value": "59216-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "WechatAPI.testcase.test_work"}]}